src/appengine/handlers/cron/fuzzer_and_job_weights.py

```diff
@@ -20,19 +20,32 @@
 from base import utils
 from datastore import data_handler
 from datastore import data_types
+from datastore import ndb
```

Comment:
Is there any advantage to making these changes in this file rather than a new one? I don't feel too strongly if you think merging them makes more sense, but I didn't see anything obvious at a glance.